code/modules/mob/living/carbon/xenomorph/castes/hivemind/castedatum_hivemind.dm

```diff
@@ -110,11 +98,4 @@
 	// *** Evolution *** //
 	upgrade_threshold = 1000
 
-	// *** Abilities *** //
-	actions = list(
-		/datum/action/xeno_action/plant_weeds/slow,
-		/datum/action/xeno_action/activable/psychic_cure,
-		/datum/action/xeno_action/choose_resin,
-		/datum/action/xeno_action/toggle_pheromones,
-		/datum/action/xeno_action/activable/secrete_resin/slow,
-		)
```

Comment:
And those. I'll dm you a solution to the psychic cure which can nerf them but not remove them.